atecc608a_se.c

```diff
@@ -161,6 +161,43 @@ static psa_status_t atecc608a_export_public_key(psa_key_slot_number_t key,
     atecc608a_deinit();
     return status;
 }
+static psa_status_t atecc608a_import_public_key(psa_key_slot_number_t key_slot,
+                                                psa_key_lifetime_t lifetime,
+                                                psa_key_type_t type,
+                                                psa_algorithm_t alg,
+                                                psa_key_usage_t usage,
+                                                const uint8_t *p_data,
+                                                size_t data_length)
+{
+    const uint16_t key_id = key_slot;
+    psa_status_t status = PSA_ERROR_GENERIC_ERROR;
+
+    /* Keys 8 to 15 can store public keys. Slots 1-7 are too small. */
+    if (key_id < 7 || key_id > 15 || data_length != ATCA_PUB_KEY_SIZE)
```

Comment:
This isn't device configuration specific, right? This is an inherent property of the device itself, no matter how the device is configured.

Reply:
Yes - it's about the slot sizes - slots 0-7 have 32 bytes of size, 9-15 - 72, slot 8 - 416.